assessments/projects/openkruise/self-assessment.md

```diff
@@ -0,0 +1,656 @@
+# OpenKruise Self-assessment
+
+The Self-assessment is the initial document for projects to begin thinking
```

Comment:
I've gone through the document and removed all template text. There is one bit of text in the document under the "Self-assessment use" section which I've decided to keep as it gives a bit more context on the document itself. It will be common across all the self-assessments but I felt this context still made sense to keep.